library/ZendService/Twitter/Response.php

```diff
@@ -152,4 +160,52 @@ public function toValue()
     {
         return $this->jsonBody;
     }
+
+    /**
+     * Retun the array of response rate info
+     *
+     * @return array
+     */
+    public function getRateLimit()
```

Comment:
This method is usually not required. If it is required, then you should really not return an array, but rather a value object that represents the data-structure for rate limiting